v3/generator/mapping.go

```diff
@@ -12,13 +29,6 @@ var APIMap = Map{
 				{Name: "Get", OAPIName: "GetSshKey"},
 			},
 		},
-		{
-			RootName: "InstanceTypes",
-			Fns: []Fn{
-				{Name: "List", OAPIName: "ListInstanceTypes"},
-				{Name: "Get", OAPIName: "GetInstanceType"},
-			},
-		},
```

Comment:
Why was it necessary to remove this?

Reply:
It was not removed, it was just moved to make each key sorted